alerts/openshift-container-storage-operator/CephMgrIsAbsent.md

```diff
@@ -0,0 +1,45 @@
+# CephMgrIsAbsent
+
+## Meaning
+
+Ceph Manager cannot be found or accessed from Prometheus target discovery.
+
+## Impact
+
+Storage metrics collector won't be available and users won't be able to see
+the storage metrics anymore. Not having a Ceph manager running impacts the
+monitoring of the cluster, PVC creation and deletion requests and should be
+resolved as soon as possible. Impact is critical here.
+
+## Diagnosis
+
+### Check if it is a network issue
+Check the network connectivity by following the steps in this [SOP](check-ceph-network-connectivity.md). 
```

Comment:
The helper file name is wrong, and use relative paths in the links, in this way:

`Check the [network connectivity](helpers/networkConnectivity.md) `

Reply:
Done